src/components/common/animation-control/animation-slider.js

```diff
@@ -141,9 +136,22 @@ const AnimationControlFactory = () => {
       this.setState({isAnimating: false});
     };
 
+    toggleSpeedControl = () => {
+      this.setState({showSpeedControl: !this.state.showSpeedControl});
+    };
+
+    _updateSpeed = speed => {
```

Comment:
this function is redundant
You can just directly pass it to SpeedControl
```js
updateAnimationSpeed={this.props.updateSpeed}
```
Also why the actual action is called `updateAnimationSpeed` in `visState`, but the prop in `AnimationSlider` here is `updateSpeed` and the prop in `SpeedControl` is `updateAnmationSpeed`? Please name them consistently

Reply:
for trip side it's now called updateLayerAnimationSpeed. for the filter side kept it the same as the previous to differentiate the 2 reducers